File: Data Structures/Trees/Preorder Traversal.java
/* Contributer : github.com/Shaurya9619
	   Email : shaurya96gairola@gmail.com
	*/
static void preOrder (Node root){
        if (root!= null){
        System.out.print(root.data + " ");
        preOrder(root.left);
        preOrder(root.right);
        }}

File: Data Structures/Trees/Preorder Traversal.cpp
/* Contributer : github.com/Shaurya9619
	   Email : shaurya96gairola@gmail.com
	*/
void preOrder(node* root){


    if (root != NULL)
    {
        cout <<root->data<<" ";
        preOrder(root->left);
        preOrder(root->right);
    }
}
